test/tools/isoltest.cpp

```diff
@@ -175,8 +199,7 @@ TestTool::Request TestTool::handleResponse(bool const _exception)
 			}
 		case 'e':
 			cout << endl << endl;
-			if (system((TestTool::editor + " \"" + m_path.string() + "\"").c_str()))
-				cerr << "Error running editor command." << endl << endl;
+			runEditor(m_path);
 			return Request::Rerun;
```

Comment:
Should this abort if running the editor failed?

Reply:
@axic Since it's an interactive session and the should immediately notice a failing editor run *and* we're returning with `Rerun` (maybe it's working on the second time), I don't think we should be aborting on any failure case, the user can opt for abort on the next loop iteration himself.